src/app/shared/services/ros-service/ros-mock.service.ts

```diff
@@ -124,6 +125,24 @@ export class RosService implements IRosService {
         }, 1000);
     }
 
+    checkTokenExists(): Observable<ExistTokenResponse> {
+        return new BehaviorSubject({token_exists: true, token_active: true});
+    }
+
+    deleteTokenMessage() {
+        return;
+    }
+
+    encryptToken(token: string, password: string): Observable<boolean> {
+        console.info(JSON.stringify({token: token, password: password}));
+        return new BehaviorSubject(true);
+    }
+
+    decryptToken(password: string): Observable<boolean> {
+        console.info(JSON.stringify({password: password}));
+        return new BehaviorSubject(true);
+    }
```

Comment:
should be tested

Reply:
I added some tests, except for `deleteTokenMessage`, since it's not doing anything.